### DISCUSSION TITLE
Bug Report: KE compiles are slow on my computer

### DISCUSSION THREAD

**Discussion by aflacc** [KadeArchive/Kade-Engine#568] [Q&A]
Okay, so I know I said this awhile ago, but then it was really just a joke. (#194) But I genuinely would like to actually know what's going on here. Please do not close this one because I want to know whats causing it to do this for me.

Ok so basically before I explain anything about my computer, I just wanna explain whats happening. Its not *every* Kade Engine build that does this, its just the ones that come from my computer (as in I compiled it on my computer). I am able to play any mods that are on Kade Engine without this issue, but compiles from my computer are slow. I think the [vs Kapi mod](https://gamebanana.com/mods/44683) also has this issue too. The whole game is super slow until you load up a song. I think the KadeEngine build of [vs Aflac](https://gamebanana.com/mods/55720) does this too, but I haven't seen anyone actually bring it up even with the Kade version having three times the ProjectFNF one.

So I haven't actually tried this again with the latest latest build of KE, and I noticed that both mods that do this used the Versions before the ModChart support (I think I had KE downloaded already so I just used that for the vs aflac port). My computer is a 64x Windows X (I like to call Windows 10 that)

What I am thinking could possibly be the cause would be something in the code for KE that handles something that ends once you load a song up.

**Comment by isakube** [comment]
get a better pc nerd /j

**Comment by ghost** [comment]
@aflacc Do you mean that the animations are 0.5x speed until you click on a song?
get that with most web versions of FNF mods

**Comment by aflacc** [comment]
> @aflacc Do you mean that the animations are 0.5x speed until you click on a song?
> get that with most web versions of FNF mods

yeah but thats on the windows builds too

**Comment by aflacc** [comment]
Kade. y this happen
